[🐛 Bug]: Setup new TS WDIO project with Configuration Helper results in broken 'Test Files'

### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md#reporting-new-issues).

### WebdriverIO Version

Latest V8

### Node.js Version

14.17.6

### Mode

Standalone Mode

### Which capabilities are you using?

```typescript
capabilities: [{
        maxInstances: 5,
        //
        browserName: 'chrome',
        acceptInsecureCerts: true
}],
```


### What happened?

Installing a fresh clean project:
WDIO Configuration Helper

? Where is your automation backend located? On my local machine
? Which framework do you want to use? cucumber
? Do you want to use a compiler? TypeScript (https://www.typescriptlang.org/)
? Where are your feature files located? ./features//*.feature
? Where are your step definitions located? ./features/step-definitions/steps.ts
? Do you want WebdriverIO to autogenerate some test files? Yes
? Do you want to use page objects (https://martinfowler.com/bliki/PageObject.html)? Yes
? Where are your page objects located? ./features/pageobjects//*.ts
? Which reporter do you want to use? spec, junit
? Do you want to add a service to your test setup? selenium-standalone
? What is the base url? http://localhost

Installing wdio packages:

    @wdio/local-runner
    @wdio/cucumber-framework
    @wdio/spec-reporter
    @wdio/junit-reporter
    @wdio/selenium-standalone-service
    ts-node
    typescript

added 163 packages, and audited 469 packages in 22s

42 packages are looking for funding
run npm fund for details

found 0 vulnerabilities

**So after this basic config helper setup I did:**

`npm run wdio`

```
[0-0] 2021-11-11T11:10:22.273Z ERROR @wdio/runner: TSError: ⨯ Unable to compile TypeScript:
[0-0] features/pageobjects/login.page.ts(1,41): error TS1109: Expression expected.
[0-0] features/pageobjects/login.page.ts(1,42): error TS1127: Invalid character.
[0-0] features/pageobjects/login.page.ts(1,43): error TS1005: ';' expected.
[0-0] features/pageobjects/login.page.ts(1,58): error TS1127: Invalid character.
[0-0] features/pageobjects/login.page.ts(1,59): error TS1005: ';' expected.
[0-0] features/pageobjects/login.page.ts(12,60): error TS1434: Unexpected keyword or identifier.
[0-0] features/pageobjects/login.page.ts(12,67): error TS1068: Unexpected token. A constructor, method, accessor, or property was expected.
[0-0] features/pageobjects/login.page.ts(12,94): error TS1109: Expression expected.
[0-0] features/pageobjects/login.page.ts(16,5): error TS1128: Declaration or statement expected.
[0-0] features/pageobjects/login.page.ts(16,12): error TS1434: Unexpected keyword or identifier.
[0-0] features/pageobjects/login.page.ts(16,31): error TS1005: ';' expected.
[0-0] features/pageobjects/login.page.ts(16,94): error TS1109: Expression expected.
[0-0] features/pageobjects/login.page.ts(20,5): error TS1128: Declaration or statement expected.
[0-0] features/pageobjects/login.page.ts(20,12): error TS1434: Unexpected keyword or identifier.
[0-0] features/pageobjects/login.page.ts(20,27): error TS1005: ';' expected.
```
Checking those files:
```
import { ChainablePromiseElement } from &#39;webdriverio&#39;;

import Page from './page';

/**
 * sub page containing specific selectors and methods for a specific page
 */
class SecurePage extends Page {
    /**
     * define selectors using getter methods
     */
    public get flashAlert(): ChainablePromiseElement&lt;Promise&lt;WebdriverIO.Element&gt;&gt; {
        return $('#flash');
    }
}
```
All the code consists of HTML code, instead of the correct quotes ' or ", also < and >


### What is your expected behavior?

That all the files have the correct 
`' or "`

`< or >`

etc.

### How to reproduce the bug.

To reproduce:
Installing packages:  @wdio/cli


added 305 packages, and audited 306 packages in 27s

36 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

Finished installing packages.

Running WDIO CLI Wizard...

=========================
WDIO Configuration Helper
=========================

? Where is your automation backend located? On my local machine
? Which framework do you want to use? cucumber
? Do you want to use a compiler? TypeScript (https://www.typescriptlang.org/)
? Where are your feature files located? ./features/**/*.feature
? Where are your step definitions located? ./features/step-definitions/steps.ts
? Do you want WebdriverIO to autogenerate some test files? Yes
? Do you want to use page objects (https://martinfowler.com/bliki/PageObject.html)? Yes
? Where are your page objects located? ./features/pageobjects/**/*.ts
? Which reporter do you want to use? spec, junit
? Do you want to add a service to your test setup? selenium-standalone
? What is the base url? http://localhost

Installing wdio packages:
- @wdio/local-runner
- @wdio/cucumber-framework
- @wdio/spec-reporter
- @wdio/junit-reporter
- @wdio/selenium-standalone-service
- ts-node
- typescript

added 163 packages, and audited 469 packages in 22s

42 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

`npm run wdio`

### Relevant log output

```typescript
Please read my description
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

--- a/packages/wdio-cli/src/templates/exampleFiles/pageobjects/login.page.js.ejs
+++ b/packages/wdio-cli/src/templates/exampleFiles/pageobjects/login.page.js.ejs
@@ -1,5 +1,5 @@
-<%= isUsingTypeScript ? "import { ChainablePromiseElement } from 'webdriverio';" : "" %>
-    
+<%- isUsingTypeScript ? "import { ChainablePromiseElement } from 'webdriverio';" : "" %>
+
 <%- isUsingTypeScript || isUsingBabel
     ? "import Page from './page';"
     : "const Page = require('./page');" %>
@@ -11,15 +11,15 @@ class LoginPage extends Page {
     /**
      * define selectors using getter methods
      */
-    <%= isUsingTypeScript ? "public " : "" %>get inputUsername()<%= isUsingTypeScript ? ": ChainablePromiseElement<Promise<WebdriverIO.Element>>" : "" %> {
+    <%- isUsingTypeScript ? "public " : "" %>get inputUsername()<%- isUsingTypeScript ? ": ChainablePromiseElement<Promise<WebdriverIO.Element>>" : "" %> {
         return $('#username');
     }
-    
-    <%= isUsingTypeScript ? "public " : "" %>get inputPassword()<%= isUsingTypeScript ? ": ChainablePromiseElement<Promise<WebdriverIO.Element>>" : "" %> {
+
+    <%- isUsingTypeScript ? "public " : "" %>get inputPassword()<%- isUsingTypeScript ? ": ChainablePromiseElement<Promise<WebdriverIO.Element>>" : "" %> {
         return $('#password');
     }
-    
-    <%= isUsingTypeScript ? "public " : "" %>get btnSubmit()<%= isUsingTypeScript ? ": ChainablePromiseElement<Promise<WebdriverIO.Element>>" : "" %> {
+
+    <%- isUsingTypeScript ? "public " : "" %>get btnSubmit()<%- isUsingTypeScript ? ": ChainablePromiseElement<Promise<WebdriverIO.Element>>" : "" %> {
         return $('button[type="submit"]');
     }
 
@@ -27,7 +27,7 @@ class LoginPage extends Page {
      * a method to encapsule automation code to interact with the page
      * e.g. to login using username and password
      */
-    <%= isUsingTypeScript ? "public " : "" %>async login (username<%= isUsingTypeScript ? ": string": "" %>, password<%= isUsingTypeScript ? ": string": "" %>)<%= isUsingTypeScript ? ": Promise<void>" : "" %> {
+    <%- isUsingTypeScript ? "public " : "" %>async login (username<%- isUsingTypeScript ? ": string": "" %>, password<%- isUsingTypeScript ? ": string": "" %>)<%- isUsingTypeScript ? ": Promise<void>" : "" %> {
         await this.inputUsername.setValue(username);
         await this.inputPassword.setValue(password);
         await this.btnSubmit.click();
@@ -36,9 +36,9 @@ class LoginPage extends Page {
     /**
      * overwrite specific options to adapt it to page object
      */
-    <%= isUsingTypeScript ? "public " : "" %>open()<%= isUsingTypeScript ? ": Promise<string>" : "" %> {
+    <%- isUsingTypeScript ? "public " : "" %>open()<%- isUsingTypeScript ? ": Promise<string>" : "" %> {
         return super.open('login');
     }
 }
 
-<%= isUsingTypeScript || isUsingBabel ? "export default": "module.exports =" %> new LoginPage();
+<%- isUsingTypeScript || isUsingBabel ? "export default": "module.exports =" %> new LoginPage();

--- a/packages/wdio-cli/src/templates/exampleFiles/pageobjects/page.js.ejs
+++ b/packages/wdio-cli/src/templates/exampleFiles/pageobjects/page.js.ejs
@@ -2,12 +2,12 @@
 * main page object containing all methods, selectors and functionality
 * that is shared across all page objects
 */
-<%= isUsingTypeScript || isUsingBabel ? "export default" : "module.exports =" %> class Page {
+<%- isUsingTypeScript || isUsingBabel ? "export default" : "module.exports =" %> class Page {
     /**
     * Opens a sub page of the page
     * @param path path of the sub page (e.g. /path/to/page.html)
     */
-    <%= isUsingTypeScript ? "public " : "" %>open(path<%= isUsingTypeScript ? ": string" : "" %>)<%= isUsingTypeScript ? ": Promise<string>" : "" %> {
+    <%- isUsingTypeScript ? "public " : "" %>open(path<%- isUsingTypeScript ? ": string" : "" %>)<%- isUsingTypeScript ? ": Promise<string>" : "" %> {
         return browser.url(`https://the-internet.herokuapp.com/${path}`)
     }
 }

--- a/packages/wdio-cli/src/templates/exampleFiles/pageobjects/secure.page.js.ejs
+++ b/packages/wdio-cli/src/templates/exampleFiles/pageobjects/secure.page.js.ejs
@@ -1,4 +1,4 @@
-<%= isUsingTypeScript ? "import { ChainablePromiseElement } from 'webdriverio';" : "" %>
+<%- isUsingTypeScript ? "import { ChainablePromiseElement } from 'webdriverio';" : "" %>
 
 <%- isUsingTypeScript || isUsingBabel
     ? "import Page from './page';"
@@ -11,9 +11,9 @@ class SecurePage extends Page {
     /**
      * define selectors using getter methods
      */
-    <%= isUsingTypeScript ? "public " : "" %>get flashAlert()<%= isUsingTypeScript ? ": ChainablePromiseElement<Promise<WebdriverIO.Element>>" : "" %> {
+    <%- isUsingTypeScript ? "public " : "" %>get flashAlert()<%- isUsingTypeScript ? ": ChainablePromiseElement<Promise<WebdriverIO.Element>>" : "" %> {
         return $('#flash');
     }
 }
 
-<%= isUsingTypeScript || isUsingBabel ? "export default": "module.exports =" %> new SecurePage();
+<%- isUsingTypeScript || isUsingBabel ? "export default": "module.exports =" %> new SecurePage();

--- a/packages/wdio-cli/src/templates/wdio.conf.tpl.ejs
+++ b/packages/wdio-cli/src/templates/wdio.conf.tpl.ejs
@@ -6,11 +6,11 @@
     //<%
 
     if(answers.expEnvAccessKey){ %>
-    hostname: '<%= answers.expEnvHostname %>',
+    hostname: '<%- answers.expEnvHostname %>',
     <% if (answers.expEnvPort === '443'){%>protocol: 'https',
     <%} else if (answers.expEnvPort === '80'){%>protocol: 'http',
-    <%} else { %>protocol: '<%= answers.expEnvProtocol %>',<%}%>
-    port: <%= answers.expEnvPort %>,
+    <%} else { %>protocol: '<%- answers.expEnvProtocol %>',<%}%>
+    port: <%- answers.expEnvPort %>,
     path: '/wd/hub',
     <% } else if(answers.hostname && answers.port) { %>
     //
@@ -24,9 +24,9 @@
     // according to your user and key information. However, if you are using a private Selenium
     // backend you should define the host address, port, and path here.
     //
-    hostname: '<%= answers.hostname %>',
-    port: <%= answers.port %>,
-    path: '<%= answers.path %>',<% } else if (answers.services.includes('appium')) { %>
+    hostname: '<%- answers.hostname %>',
+    port: <%- answers.port %>,
+    path: '<%- answers.path %>',<% } else if (answers.services.includes('appium')) { %>
     port: 4723,<% }
 
     // Protocol: http | https
@@ -40,8 +40,8 @@
     // should work too though). These services define specific user and key (or access key)
     // values you need to put in here in order to connect to these services.
     //
-    user: process.env.<%= answers.env_user %>,
-    key: process.env.<%= answers.env_key %>,
+    user: process.env.<%- answers.env_user %>,
+    key: process.env.<%- answers.env_key %>,
     //
     // If you run your tests on Sauce Labs you can specify the region you want to run your tests
     // in via the `region` property. Available short handles for regions are `us` (default), `eu` and `apac`.
@@ -49,7 +49,7 @@
     // If you don't provide the region it will default for the `us`
     <% }
     if(answers.headless) { %>headless: true,<% }
-    if(answers.region) { %>region: '<%= answers.region %>',<% } %>
+    if(answers.region) { %>region: '<%- answers.region %>',<% } %>
     //
     // ==================
     // Specify Test Files
@@ -67,7 +67,7 @@
     // will be called from there.
     //
     specs: [
-        '<%= answers.specs %>'
+        '<%- answers.specs %>'
     ],
     // Patterns to exclude.
     exclude: [
@@ -96,7 +96,7 @@
     // https://saucelabs.com/platform/platform-configurator
     //
     capabilities: [{
-    <%if(answers.expEnvAccessKey){%> 'experitest:accessKey': '<%= answers.expEnvAccessKey %>',<%}%>
+    <%if(answers.expEnvAccessKey){%> 'experitest:accessKey': '<%- answers.expEnvAccessKey %>',<%}%>
         // maxInstances can get overwritten per capability. So if you have an in-house Selenium
         // grid with only 5 firefox instances available you can make sure that not more than
         // 5 instances get started at a time.
@@ -140,7 +140,7 @@
     // with `/`, the base url gets prepended, not including the path portion of your baseUrl.
     // If your `url` parameter starts without a scheme or `/` (like `some/path`), the base url
     // gets prepended directly.
-    baseUrl: '<%= answers.baseUrl %>',
+    baseUrl: '<%- answers.baseUrl %>',
     //
     // Default timeout for all waitFor* commands.
     waitforTimeout: 10000,
@@ -167,7 +167,7 @@
     //
     // Make sure you have the wdio adapter package for the specific framework installed
     // before running any tests.
-    framework: '<%= answers.framework %>',
+    framework: '<%- answers.framework %>',
     //
     // The number of times to retry the entire specfile when it fails as a whole
     // specFileRetries: 1,
@@ -207,7 +207,7 @@
     // If you are using Cucumber you need to specify the location of your step definitions.
     cucumberOpts: {
         // <string[]> (file/dir) require files before executing features
-        require: ['<%= answers.stepDefinitions %>'],
+        require: ['<%- answers.stepDefinitions %>'],
         // <boolean> show full backtrace for errors
         backtrace: false,
         // <string[]> ("extension:module") require files with the given EXTENSION after requiring MODULE (repeatable)

